Unrelated build failure: `12:09:32 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/lib/ClangImporter/ClangImporter.cpp:769:23: error: no matching constructor for initialization of 'clang::Preprocessor'`

--- a/validation-test/compiler_crashers_2_fixed/sr10108.swift
+++ b/validation-test/compiler_crashers_2_fixed/sr10108.swift
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -typecheck %s
+// RUN: not %target-swift-frontend -typecheck %s
 
 // Just make sure we don't crash.
 

